web.browser/src/containers/App/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+
+import Categories from '../../containers/Categories';
+
+import styles from './styles.css';
```

Comment:
New lines are weird here. Suggest:

```js
import React, { Component } from 'react';
import PropTypes from 'prop-types';

import Categories from '../../containers/Categories';
import styles from './styles.css';
```

This way you're separating libraries from your own code.